.circleci/config.base.yml

```diff
@@ -1249,11 +1278,8 @@ workflows:
                 - /tagged-release\/.*/
                 - /run-e2e\/.*/
 
-      - deploy:
+      - publish_to_npm:
```

Comment:
Should this step include ```verify-pkg-binaries``` based on the workflow of ```taggedrelease``` ?

Reply:
No `verify-pkg-binaries` was intentionally left as leaf node in all workflows, so that it does not block release should we decide to ignore it. It's main purpose is to catch problems at PR stage.

See recent release for reference. https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/20407/workflows/083e1f45-095b-47e5-8465-241f1968bf35